.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
```

Comment:
This means it'll automatically trigger when we save a draft of a new release? Or do we specifically have to choose the "pre-release" box?

Reply:
See the third info box here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

```Note: The prereleased type will not trigger for pre-releases published from draft releases, but the published type will trigger. If you want a workflow to run when stable and pre-releases publish, subscribe to published instead of released and prereleased.```

So this will trigger when you draft a release or a prerelease, but not a draft. That's intentional, because drafts would have access to all the same information via the branch tags, just not uploaded to the "Release" section.

I'll clarify this in the main PR comment.

Reply:
Cool, ok. The only thing I'm confused about is how you supposedly created a [`test-auto-release`](https://github.com/BrightSpots/rcv/releases/tag/test-auto-release) tag, but I don't see it in any of the branches when I run `git tag`. Know why this is? Does it hold off on pushing the tag until the non-pre-release is published?

Reply:
I created several test tags and then deleted them, so `test-auto-release` was a tag I created and deleted while debugging the workflow